src/pz390.java

```diff
@@ -2788,10 +2798,16 @@ private void ins_lt_c0() {
 		 */
 		switch (opcode1) {
 		case 0x80: // 1530 "8000" "SSM" "S"
+             if (psw_problem_state == psw_problem_mode)
+                {set_psw_check(psw_pic_priv); // RPI 1622 DK Privileged operation exception
+                 } // in problem state issue privileged-operation exception, otherwise no-op until we implement this instruction
```

Comment:
Code looks okay. Suggestion. Since psw_problem_state is a boolean, you could replace the if statement with
```
if (psw_problem_state)
```

Reply:
I understand your suggestion. However, I do prefer an explicit test. To me, that is more readable and it lowers the risk of confusion as I don't have to remember the semantics of 'true' versus 'false'.